core/sass/partials/_forms.scss

```diff
@@ -19,6 +19,9 @@
 // @mixin body-s
 // @mixin body-m
 
+#content .form-group input[type=checkbox] {
```

Comment:
can you move this into `contact.scss`. seems to belong with the other styling for the contact forms rather than generic form styling